src/Symfony/Component/DependencyInjection/Container.php

```diff
@@ -66,6 +66,7 @@ class Container implements ResettableContainerInterface
     protected $services = array();
     protected $methodMap = array();
     protected $privates = array();
+    protected $synthetics = array();
```

Comment:
Could be, perhaps ideally we should just deprecate this method and dont care :) it can be 
interpreted in different ways.